js/ui/text_box/text_box.js

```diff
@@ -84,10 +84,6 @@ var TextBox = TextEditor.inherit({
     },
 
     _toggleMaxLengthProp: function() {
-        if(this._isAndroidOrIE()) {
```

Comment:
Check [T751220](https://supportcenter.devexpress.com/internal/ticket/details/T751220) pls

Reply:
I am aware of this ticket. And I am not able to reproduce this behaviour now, with my fix.